README.md

```diff
@@ -1241,23 +1244,28 @@ and the Dfinity project. [\[18\]][18]
 The Lightning Network is a proposed message relay network operating at a layer
 above the Bitcoin blockchain, enabling many orders of magnitude improvement in
 transaction throughput by moving the majority of transactions outside of the
-consensus ledger into so-called "payment channels". This is made possible (with
-great difficulty) by the Bitcoin scripting language, which enables parties to
+consensus ledger into so-called "payment channels".
+<!---
+lightning network exists beyond bitcoin
+--->
+Which enables parties to
 enter into stateful contracts where the state can be updated by sharing digital
 signatures, and contracts can be closed by finally publishing evidence onto the
 blockchain, a mechanism first popularized by cross-chain atomic swaps.  By
 openning payment channels with many parties, participants in the Lightning
-Network can become focal points for routing the payments of others, leading to a
-fully connected payment channel network, at the cost of capital being tied up on
-payment channels.
-
-While the Lightning Network can also easily extend across multiple independent
-blockchains to allow for the transfer of _value_ via an exchange market, it
-cannot be used to assymetrically transfer _tokens_ from one blockchain to
-another.  The main benefit of the GnuClear network described here is to enable
-such direct token transfers.  That said, we expect payment channels and the
-Lightning Network to become widely adopted along with our token transfer
-mechanism, for cost-saving and privacy reasons.
+Network can become focal points for routing the payments, and contracts of others, leading to a
+fully connected channel network, at the cost of capital being tied up on
+payment channels. This cost can be completely removed by making it into
```

Comment:
> It will be too many bonds for stopping double-signing, so you will waste money by the interest rate.

Assume 0 in the reserve-pool.  There's no money wasted, 100% of the gnuts represent claim to fees, which arrises from the network.  Gnuts != money.  Gnuts may have high value but they can't be spent or used anywhere else, so there's no opportunity cost.

The reserve-pool tokens arguably incur opportunity cost for being bonded, but they don't decide who the validators are.  Add to the reserve-pool until marginal cost == marginal return.  There aren't "too many bonds".

> It will be too few bonds for deciding who should have consensus, so you will have retirement attacks.

A significant coalition can try to attack the system, but they will get slashed.  Those who remain will own more relative gnuts.  The system is antifragile.  But this is assuming that there will even be profitable retirement attacks.  Anyone can throttle their txs to limit exposure.